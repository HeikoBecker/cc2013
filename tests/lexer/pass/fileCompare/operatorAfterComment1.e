operatorAfterComment1.c:1:1: keyword int
operatorAfterComment1.c:1:5: identifier main
operatorAfterComment1.c:1:9: punctuator (
operatorAfterComment1.c:1:10: punctuator )
operatorAfterComment1.c:1:12: punctuator {
operatorAfterComment1.c:2:2: keyword int
operatorAfterComment1.c:2:6: identifier a
operatorAfterComment1.c:2:8: punctuator =
operatorAfterComment1.c:2:10: constant 5
operatorAfterComment1.c:2:10: constant 5
operatorAfterComment1.c:3:3: punctuator /
operatorAfterComment1.c:3:4: constant 3
operatorAfterComment1.c:3:5: punctuator ;
operatorAfterComment1.c:4:2: keyword return
operatorAfterComment1.c:4:9: constant 0
operatorAfterComment1.c:4:10: punctuator ;
operatorAfterComment1.c:5:1: punctuator }
