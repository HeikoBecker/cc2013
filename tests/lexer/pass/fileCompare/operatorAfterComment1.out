/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:1:1: keyword int
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:1:5: identifier main
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:1:9: punctuator (
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:1:10: punctuator )
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:1:12: punctuator {
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:2:2: keyword int
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:2:6: identifier a
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:2:8: punctuator =
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:2:10: constant 5
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:2:10: constant 5
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:3:3: punctuator /
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:3:4: constant 3
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:3:5: punctuator ;
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:4:2: keyword return
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:4:9: constant 0
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:4:10: punctuator ;
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/operatorAfterComment1.in:5:1: punctuator }
